packages/types/src/transaction.ts

```diff
@@ -150,6 +150,12 @@ export interface TransactionMetadata {
 
   /** Metadata for the transaction's spans, keyed by spanId */
   spanMetadata: { [spanId: string]: { [key: string]: unknown } };
+
+  /** Metadata representing information about transaction name changes  */
+  changes: TransactionNameChange[];
+
+  /** The total number of propagations that happened */
+  propagations: number;
```

Comment:
Just a train of thought: We could explore having two type definitions for all top level options. One that is user-facing, and one that is internal to us. That way we don't create technical breaking changes when we remove values like these again after testing.

Doesn't need action on your part. Just wanted to write that down.

Reply:
I think this is a great point - but it introduces some complexity where we'll need to cast appropriately when working between user-defined and internally defined types, which could cause a lot of trouble (esp in places like tests, or across package boundaries).